app/src/main/res/layout/fragment_splash.xml

```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
`move_logo` and `splash_background_image` can be exported as svg and then imported as Vector Drawables in your project.

I forgot to mention this rule of thumb in the discussion:
If an asset isn't a complex image and it looks made out of basic shapes, then you should try exporting it as svg first.